core/src/main/java/org/mskcc/cbio/oncokb/apiModels/TranscriptCoverageFilterResult.java

```diff
@@ -0,0 +1,31 @@
+package org.mskcc.cbio.oncokb.apiModels;
+
+public class TranscriptCoverageFilterResult {
+    String variant;
+    Boolean isCovered;
+
+    public TranscriptCoverageFilterResult(String variant, Boolean isCovered) {
+        this.variant = variant;
+        this.isCovered = isCovered;
+    }
+
+    public String getVariant() {
+        return this.variant;
+    }
+
+    public void setVariant(String variant) {
+        this.variant = variant;
+    }
+
+    public Boolean isIsCovered() {
```

Comment:
```suggestion
    public Boolean isCovered() {
```

Reply:
Thanks for the catch. I actually used VSCode to generate getters/setters and it generated an extra getter. There's already the getter `getIsCovered`.